src/script/calling/CallingRepository.ts

```diff
@@ -1194,7 +1194,16 @@ export class CallingRepository {
   };
 
   private getMediaStream({audio = false, camera = false, screen = false}: MediaStreamQuery, isGroup: boolean) {
-    return this.mediaStreamHandler.requestMediaStream(audio, camera, screen, isGroup);
+    return this.mediaStreamHandler.requestMediaStream(audio, camera, screen, isGroup).then(stream => {
+      return this.mediaDevicesHandler
+        .initializeMediaDevices(camera)
+        .then(() => {
+          return stream;
+        })
+        .catch(() => {
+          return stream;
+        });
+    });
   }
```

Comment:
I've modified the device list so that it is no longer initialized when the app starts. However, this makes it necessary to create it either before a call or when opening the device selection. n these two situations, a media stream will be created.